tests/end2end/data/fileselect/fileselect.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Testing fileselection</title>
+        <script type="text/javascript">
+            function onLoad() {
+                for (let id of ['#single_file', '#multiple_files']) {
+                    const input = document.querySelector(id);
+                    input.addEventListener('input', function (e) {
+                        console.log(`Files: ${JSON.stringify(input.files)}`);
```

Comment:
```suggestion
                        console.log(`Files: ${JSON.stringify(Array.from(input.files).map(f => f.name))}`);
```

there we go - still nicer than a for-loop in my opinion :slightly_smiling_face:

Reply:
Wait, actually no need for JSON at this point really:

```suggestion
                        console.log(`Files: ${Array.from(input.files).map(f => f.name).join(", ")}`);
```